typescript/src/accounts/evm/resolveViemClients.ts

```diff
@@ -132,6 +135,42 @@ export type ResolvedViemClients = {
   walletClient: WalletClient<Transport, Chain, Account>;
 };
 
+/**
+ * Get the base node RPC URL for a given network
+ *
+ * @param network - The network identifier
+ * @returns The base node RPC URL or undefined if the network is not supported
+ */
+async function getBaseNodeRpcUrl(network: string): Promise<string | undefined> {
```

Comment:
should network be typed to a union of constants? or should there be that check to see if network is supported? seems like this will return `${basePath}/rpc/v1/${network}/${json.id}` even i pass an arbitrary network?